### ISSUE TITLE
LatestVersionSubset gives wrong impression

### ISSUE THREAD

**Issue by Yalz** [SEMICeu/LinkedDataEventStreams#47]
The [LatestVersionSubset](https://w3id.org/ldes#LatestVersionSubset) gives the impression that only version (the latest) will be kept.
On [this site](https://semiceu.github.io/LinkedDataEventStreams/#version-subsets) it gives a clearer explanation that it allows multiple versions, that the amount is to be defined by the policy.

I suggest updating the https://w3id.org/ldes spec to make clear more than one version of a state object can be kept

**Comment by sandervd** [comment]
Fair point. We should also make it more clear that the retention policy determines which members can be removed from the collection, yet it doesn't give any guarantees on how and when they will disappear (a server MAY delete members).
